contrib/ci/oss-fuzz.py

```diff
@@ -0,0 +1,431 @@
+#!/usr/bin/python3
+# pylint: disable=invalid-name,missing-docstring
+#
+# Copyright (C) 2021 Richard Hughes <richard@hughsie.com>
+#
+# SPDX-License-Identifier: LGPL-2.1+
+#
+# pylint: disable=too-many-instance-attributes,no-self-use
+
+import os
+import sys
+import subprocess
+import glob
+from typing import Dict, Optional, List, Union
+
+
+class Builder:
+    def __init__(self) -> None:
+
+        self.cc = self._ensure_environ("CC", "gcc")
+        self.cxx = self._ensure_environ("CXX", "g++")
+        self.builddir = self._ensure_environ("WORK", os.path.realpath("_build"))
+        self.installdir = self._ensure_environ("OUT", os.path.realpath("_build/out"))
```

Comment:
we have `.gitignore` for `build` already, why not re-use that?

Reply:
Fair point, will fix.